Moose prompt

milljm opened the pull request:
Add pretty prompt back into the package for supported shells (sh bash zsh).
Upgrade autojump to support zsh.
Remove gnuplot module in favor of installing it via Conda.

Commit:
Remove gnuplot package and use conda

Remove gnuplot, in favor of installing it via conda.

Remove moose-tools requirement from all moose-dev-$ modules. However,
moose-tools will still be loading by default, due to gperf-tools
requirement in moose-dev-$ modules.

Create a gperf-tools advanced module so it can be loaded separately.

Commit:
Add pretty prompt for supported shells

Adding pretty prompt back into the package for supported shells.

Update autojump to support zsh (previous commit, woops).

Commit:
Use conda-forge channel

Lets use the conda-forge channel... Gnuplot is available there. As
well as most of the tools these days are more up-to-date on conda-forge.

Commit:
Separate gnuplot install from conda-forge

Okay... lets not use conda-forge for everything. Certain newer darwin
machines run into conflicts.

Commit:
Fix icecream, ccache, gcc module

Use older icecream to create a payload, then use latest for client.

Append ccache to current CC, CXX variables. That way, it is always
correct.

Export CC,CXX when using GCC module.

moosebuild commented:
All [jobs](https://civet.inl.gov/event/48676/) on b4ea8a9 : invalidated by @milljm

Upgraded CentOS and Fedora Targets to 8/31

Commit:
Iceream, RPM creation optimizations

Fix Icecream (use modified payload creator).
Delete seacas build directories (they are read only).
Speed up rpm creation.
